plugin/src/main/java/net/aufdemrand/denizen/objects/dPlayer.java

```diff
@@ -1541,6 +1538,29 @@ else if (isOnline()) {
                     .getAttribute(attribute.fulfill(1));
         }
 
+        // <--[tag]
+        // @attribute <p@player.selected_trade_index>
+        // @returns Element(Number)
+        // @description
+        // Returns the index of the trade the player is currently viewing, if any.
+        // -->
+        if (attribute.startsWith("selected_trade_index")) {
+            return new Element(((MerchantInventory) getPlayerEntity().getOpenInventory().getTopInventory())
+                    .getSelectedRecipeIndex() + 1).getAttribute(attribute.fulfill(1));
+        }

```

Comment:
What does this do when a player isn't viewing a trade?
Probably an exception. Not good.